### PR TITLE
test: transform modules in karma so sinon can stub properly

### PR DESCRIPTION
**Pull request by jrparish**
This fixes all the failing tests when running using Karma. 

https://github.com/sinonjs/sinon/issues/1711

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -33,8 +33,12 @@ module.exports = (config) => {
               path.join(__dirname, 'test'),
               require.resolve('airbnb-js-shims'),
             ],
-            query: {
-              presets: ['airbnb'],
+            options: {
+              presets: [
+                'airbnb',
+                // transform to cjs so sinon can stub properly
```
Comment (ljharb) [inline karma.conf.js]: interesting, does the airbnb preset not already do this? i think it accepts a modules option too

Comment (jrparish) [reply karma.conf.js]: I originally tried to set it on the airbnb preset, but it prevents you from specifying anything other than true/false/auto (https://github.com/airbnb/babel-preset-airbnb/blob/master/index.js#L35)

Comment (ljharb) [reply karma.conf.js]: K, I’ll have to expand that to accept cjs; in the meantime, let’s set this one to false so it doesn’t include the env preset twice?

Comment (jrparish) [reply karma.conf.js]: Sure thing 👍

**Pull request merged**